### DISCUSSION TITLE
1CH constantly opening / closing

### DISCUSSION THREAD

**Discussion by flexmatics** [nliaudat/esp32_8ch_motor_shield#29] [Q&A] [answered]
Hi!
So got 2 separate controllers. With one of those noticed that 1 channel is constantly opening / closing. Rebooting the controller helps for a day or so, but then it starts again. Also noted that this behavior can be seen for different channels - for example, after reboot, after a while it starts again, but for another channel. Attaching logs what I'm referring to (see CH1). Any advice what should I try - don't think this is intended?

![image](https://user-images.githubusercontent.com/79056776/201478201-8fc5b4a1-0e57-4d28-a896-cbb4e91f52ec.png)

**Comment by nliaudat** [comment]
perhaps thats the calibration ?
script_calibrate_valves.yaml called 1 min after boot (board.yaml)

**Comment by nliaudat** [comment, marked as answer]
that is related to https://github.com/nliaudat/floor-heating-controller firmware
Please discuss on that channel. 
Regards
